app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :posts
+  belongs_to :users
+  has_one :posts
+  validates :body, presence: true
+end
```

Comment:
- [ ] A comment belongs to a single post and belongs to a particular user. So change it belongs_to: post and belongs_to: user
- [ ] There is no need for the extra has_one:posts because the relationship is already covered in belongs_to kindly remove it